TestApp34/android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,2 +1,2 @@
-#Mon Oct 30 16:27:59 MSK 2017
 connection.project.dir=
+eclipse.preferences.version=1
```

Comment:
What is file and why do we need to check it into Git?

Reply:
If we don't need it, let's remove it.

Reply:
Fixed in 12f875e